tests/comfy_cli/command/generate/test_emit.py

```
@@ -122,15 +167,18 @@ def test_emitted_node_covers_every_widget_input(model):
     values = {"prompt": "p"}
     if ns.image_params:
         values[next(iter(ns.image_params))] = "img.png"
-    wf = emit.build_workflow(model, values)
-    inputs = wf["1"]["inputs"]
+    wf, _ops = emit.build_workflow(model, values)
+    partner = _one(wf, ns.node_class)
+    order = graph.widget_order_default(ns.node_class)
+    widget_names = {name for name in order if name in graph.widget_defaults(ns.node_class) or True}
```

Pylint (4.0.7)

[refactor] Boolean condition 'name in graph.widget_defaults(ns.node_class) or True' will always evaluate to 'True'

(R1727)